test_rclcpp/test/test_multithreaded.cpp

```diff
@@ -50,7 +50,7 @@ static inline void multi_consumer_pub_sub_test(bool intra_process)
     [&counter, &intra_process](test_rclcpp::msg::UInt32::ConstSharedPtr msg,
       const rmw_message_info_t & info) -> void
     {
-      ++counter;
+      counter.fetch_add(1);
```

Comment:
Ah, very good to know! I'll leave them as is here, but will remember in the future that the shorter syntax is just as good.